roles/update/molecule/default/converge.yml

```diff
@@ -20,5 +20,13 @@
   vars:
     ansible_user_dir: "{{ lookup('env', 'HOME') }}"
     cifmw_update_run_dryrun: true
-  roles:
-    - role: "update"
+    cifmw_openshift_kubeconfig: "{{ lookup('env', 'HOME') }}/.kube/config"
```

Comment:
not so sure here.
I think it should be using same value as in other roles: `cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"`.
The `.kube/.config might not exists`